pages/saved.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Image from 'next/image'
+import { FaArrowRight } from "react-icons/fa";
+import Link from 'next/link';
+
+const Saved = () => {
+  return (
+    <>
+      <div className="p-8">
+        <h1 className='text-2xl font-medium'>Saved</h1>
+        <p className='text-[#A6ABBF]'>Your saved resources, ready when you are.</p>
+      </div>
+      <div className='flex h-screen justify-center items-center flex-col'>
+        <Image src="/SaveRemove.png" width={120} height={120} alt={''} />
+        <h1 className='text-center sm:text-center md:text-center mt-5 text-lg font-medium'>Your collection is empty! Start saving items for later reference.</h1>
```

Comment:
If we need the text to be always centered let's increase the specificity for the class.

```suggestion
        <h1 className='mt-5 text-lg font-medium !text-center'>Your collection is empty! Start saving items for later reference.</h1>
```